src/privateStore.js

```diff
@@ -61,7 +64,13 @@ class PrivateStore {
    * @return    {Boolean}                           true if successful
    */
   async remove (key) {
-    return this.set(key, null)
+    if (!this.db) throw new Error('_sync must be called before interacting with the store')
+
+    const dbKey = this._genDbKey(key)
+    const hash = await this.db.del(dbKey)
+    this.updateRoot(hash)
+
+    return true
```

Comment:
Isn't strictly part of this story, but can you change the last three lines to 
```js
return this.updateRoot(hash)
```
And please do the same thing for the `set` method :)

Reply:
Yeah, this is nice. I'll make this change.